kuksa-sdk/src/test/kotlin/org/eclipse/kuksa/DataBrokerConnectionTest.kt

```diff
@@ -133,6 +136,74 @@ class DataBrokerConnectionTest : BehaviorSpec({
                 }
             }
         }
+
+        and("A Specification") {
+            val specification = VssDriver()
+            val property = Property(specification.heartRate.vssPath)
+
+            `when`("Fetching the specification") {
+
+                and("The initial value is different from the default for a child") {
+                    val newHartRateValue = 60
+                    val datapoint = Datapoint.newBuilder().setUint32(newHartRateValue).build()
+
+                    dataBrokerConnection.update(property, datapoint)
+
+                    then("Every child property has been updated with the correct value") {
+                        val updatedDriver = dataBrokerConnection.fetch(specification)
+                        val heartRate = updatedDriver.heartRate
+
+                        heartRate.value shouldBe newHartRateValue
+                    }
+                }
+            }
```

Comment:
this test is failing for me: 
`io.kotest.assertions.AssertionFailedError: expected:<60> but was:<100>
	at app//org.eclipse.kuksa.DataBrokerConnectionTest$1$1$2$1$1$1.invokeSuspend(DataBrokerConnectionTest.kt:156)
	at app//kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at app//kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at java.base@17.0.6/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@17.0.6/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base@17.0.6/java.lang.Thread.run(Unknown Source)`